src/System.Drawing.Common/tests/Drawing2D/GraphicsPathTests.cs

```diff
@@ -0,0 +1,2757 @@
+                AssertEmptyGrahicsPath(gp);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Transform_MatrixNull_ThrowsArgumentNullException()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                AssertExtensions.Throws<ArgumentNullException>("matrix", () => gp.Transform(null));
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void GetBounds_PathEmpty_ReturnsExpected()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            {
+                Assert.Equal(new RectangleF(0f, 0f, 0f, 0f), gp.GetBounds());
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void GetBounds_Rectangle_ReturnsExpected()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (Matrix matrix = new Matrix())
+            {
+                RectangleF rectangle = new RectangleF(1f, 1f, 2f, 2f);
+                gp.AddRectangle(rectangle);
+                Assert.Equal(rectangle, gp.GetBounds());
+                Assert.Equal(rectangle, gp.GetBounds(null));
+                Assert.Equal(rectangle, gp.GetBounds(matrix));
+                Assert.Equal(rectangle, gp.GetBounds(null, null));
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void GetBounds_Pie_ReturnsExpected()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (Matrix matrix = new Matrix())
+            {
+                Rectangle rectangle = new Rectangle(10, 10, 100, 100);
+                gp.AddPie(rectangle, 30, 45);
+                AssertRectangleEqual(new RectangleF(60f, 60f, 43.3f, 48.3f), gp.GetBounds(), 0.1f);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Flatten_Empty_Success()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (GraphicsPath clone = Assert.IsType<GraphicsPath>(gp.Clone()))
+            {
+                gp.Flatten();
+                Assert.Equal(gp.PointCount, clone.PointCount);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void Flatten_MatrixNull_Success()
+        {
+            using (GraphicsPath gp = new GraphicsPath())
+            using (GraphicsPath clone = Assert.IsType<GraphicsPath>(gp.Clone()))
+            {
+                gp.Flatten(null);
+                Assert.Equal(gp.PointCount, clone.PointCount);
```

Comment:
Hmm -- I see there are tests down below that do roughly that. This is okay, then.